src/expertsystem/reaction/topology.py

```diff
@@ -73,7 +73,7 @@ def edges(self) -> Dict[int, Edge]:
         return self.__edges
 
     def __repr__(self) -> str:
-        return f"{self.__class__.__name__}{(self.nodes, self.edges)}"
+        return f"{self.__class__.__name__}{(self.__nodes, self.__edges)}"
```

Comment:
I wonde what is best to do in these situations: call the internals of `self`, or use its interface (e.g. `self.nodes`, as happened previously). Same further down.

Reply:
That is true. I would say using the interface would be more consistent, so I will change that back.